build.sh

```diff
@@ -16,10 +16,10 @@
 # limitations under the License.
 #
 
-# mkdir ~/.m2
+mkdir -p ~/.m2/repository/org/keycloak
 
-# wget https://atlan-build-artifacts.s3-ap-south-1.amazonaws.com/artifact/maven_local_repository.zip
-# unzip maven_local_repository.zip -d ~/.m2
+wget https://atlan-public.s3.eu-west-1.amazonaws.com/artifact/keycloak-15.0.2.1.zip
+unzip keycloak-15.0.2.1.zip -d ~/.m2/repository/org
```

Comment:
Just tried locally it asks for replace existing jar, I think force replace is needed here with unzip command

Reply:
ignore if it  the case that ~/.m2/repository/org/keycloak will not exist

Reply:
yes it wont be there in pod, but updated it regardless.